Cargo.toml

```diff
@@ -13,23 +13,21 @@ crossbeam-channel = "0.5.8"
 parking_lot = "0.12.1"
 hashbrown = "0.14.2"
 lru = "0.12.0"
-
-# below is for the async runtime
-async-task = "4.4.0"
-futures-lite = "1.12.0"
-once_cell = "1.17.1"
-flume = "0.11.0"
+async-channel = "2.1.1"
+futures = "0.3.30"
+async-std = "1.12.0"
```

Comment:
Why are we using `async-std` instead of Tokio? We use Tokio in `surrealdb`. This way we will end up with two runtimes, instead of one.

I like `async-std`'s APIs but it's not as well maintained as Tokio. Its latest release is from over a year ago. Tokio is far more popular and a much safer bet as far as maintenance goes.

Reply:
Removed this dependency and replaced with tokio

Reply:
Removed dependency on async_std, thanks for the explanation @rushmorem :)